src/components/SidePanel/HelpComponents/HelpContent.tsx

```diff
@@ -127,7 +128,9 @@ function HelpContent({closeSidePanel}: HelpContentProps) {
                 shouldShowCloseButton={isExtraLargeScreenWidth}
             />
             {currentState === undefined ? (
-                <FullScreenLoadingIndicator style={[styles.flex1, styles.pRelative]} />
+                <View style={[styles.flex1, styles.pRelative, styles.fullScreenLoading, styles.w100]}>
```

Comment:
```suggestion
                <View style={[styles.flex1, styles.fullScreenLoading]}>
```
I think this is enough